helm/app/templates/application/app-migrate.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: app-migrate
+  annotations:
+    helm.sh/hook: "pre-upgrade"
```

Comment:
post-upgrade

Reply:
I think we also need to hook the same task from app init into app migrate

Reply:
in keycloaks case it's post-upgrade rather than pre-upgrade, because the "migrations" are api operations that need the latest version to be active

Reply:
I've done these 2 things now